bsmetadata/preprocessing_utils.py

```diff
@@ -214,10 +218,17 @@ def postprocess_entity(self, resu_list):
                     resu_list[ent][0] + resu_list[ent][1]
                 ),  # element at index = 1 in the result list corresponds to length of the entity
                 "value": entity,
+                "ent_desc": ent_desc,
```

Comment:
I would just like to draw @timoschick 's attention here as it concerns the final data format.

For the record, HTML metadata also has an additional key called [`html_attrs`](https://github.com/bigscience-workshop/metadata/blob/ecae719716384caa51d1cd564078ea0fb807da95/bsmetadata/preprocessing_tools/html_parser/objects.py#L53)